src/scripts/main.js

```diff
@@ -358,3 +358,24 @@ const people = [
 console.log(people); // you can remove it
 
 // write your code here
+const elemTable = document.querySelector('.dashboard');
+
+for (const person of people) {
+  const row = document.createElement('tr');
+
+  [
+    person.name,
+    person.sex === 'm' ? 'Male' : 'Female',
+    person.born,
+    person.died,
+    person.died - person.born,
+    Math.ceil(person.born / 100),
+  ].forEach((data) => {
+    const cell = document.createElement('td');
+
+    cell.append(data);
+    row.append(cell);
+  });
+
+  elemTable.firstElementChild.append(row);
```

Comment:
Ensure that `elemTable.firstElementChild` is a valid table element (like `<tbody>` or `<thead>`). If `firstElementChild` is `null` or not a valid table element, this will cause an error. Consider checking if `firstElementChild` is not `null` and is a valid table element before appending rows.